api/runes/src/env.ts

```diff
@@ -9,17 +9,20 @@ const schema = Type.Object({
   /** Port in which to serve the Admin RPC interface */
   ADMIN_RPC_PORT: Type.Number({ default: 3001, minimum: 0, maximum: 65535 }),
 
-  RUNES_PGHOST: Type.String(),
+  RUNES_PGHOST: Type.String({ default: 'localhost' }),
   RUNES_PGPORT: Type.Number({ default: 5432, minimum: 0, maximum: 65535 }),
-  RUNES_PGUSER: Type.String(),
-  RUNES_PGPASSWORD: Type.String(),
-  RUNES_PGDATABASE: Type.String(),
+  RUNES_PGUSER: Type.String({ default: 'postgres' }),
+  RUNES_PGPASSWORD: Type.String({ default: 'password' }),
+  RUNES_PGDATABASE: Type.String({ default: 'postgres' }),
```

Comment:
Instead of defining these to avoid CI errors you should define the ENV vars in `ci.yaml` inside the `test` job

Reply:
Thanks, feel free to merge